### PR TITLE
[WIP] Removed Windows-specific bluetooth overrides

### PR DESCRIPTION
**Pull request by jdpigeon**
I attempted to get the windows muse update working, but it looks like it won't be possible until we update the version of electron that the app uses. The app is on Electron 2, which uses Chrome 60. Chrome is now at 80 (windows web bluetooth was added in 70) and Electron is now at version 8(!)

### CODE REVIEW

**Comment by jdpigeon** [comment]
That said, I believe these are the correct js-side changes

**Comment by teonbrooks** [comment]
let me know when you would like eyes on this

**Comment by teonbrooks** [comment]
just curious, what is state of this PR?

**Comment by jdpigeon** [comment]
This is blocked until someone spends the time to update the electron version and deal with how it breaks our electron-boilerplate project setup.

The most proximal reason why this is hard is that the boilerplate this project forks (https://github.com/electron-react-boilerplate/electron-react-boilerplate) switched from a two package.json to a single package.json setup since we published the app. Because we use a lot of native dependencies for bluetooth and such, we leaned into the two package.json structure pretty hard and I'm afraid that it'll take some work to rewrite our build system with a one package.json structure.

However, updating our dependencies would be amazing for this project. It's also possible that we could break off of the boilerplate entirely and create a simpler (and hopefully more maintainable) project setup.

**Comment by jdpigeon** [comment]
Also, the prospect that Muse is going to be the ideal educational neuroscience hardware is becoming less and less likely 😅

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,105 +1,130 @@
 # 0.13.0 (May 18, 2020) - Improved usability and tasks
```
Comment (teonbrooks) [inline CHANGELOG.md:1]: @jdpigeon, could you update the changelog when you get a chance

**Pull request merged**